src/reference/config/models/.partials/cataloged.md

```diff
@@ -0,0 +1,44 @@
+### catalog\*
+
+> `object`
+
+To map defined catalog for schema retrieval based on catalog specific parameters. Any of the possible combination can be configured.
+
+> `id`
+-----
+> `strategy`
+> `version`
+-----
+> `subject`
+> `version`
+-----
+> `schema`
+> `version`
```

Comment:
I think this is an `engine.schema.json` bug, so we'll likely need to remove this.

Early on there was some confusion between the terminology for `schema` vs `subject` so we ended up with both briefly, but have since consolidated on `subject`, also `schema` seems to be gone from the code.

Reply:
Got it, I will make the change in both places